moto/dynamodb2/models.py

```diff
@@ -285,6 +285,9 @@ def __init__(self, hash_key, hash_key_type, range_key, range_key_type, attrs):
     def __repr__(self):
         return "Item: {0}".format(self.to_json())
 
+    def size(self):
+        return sum([bytesize(key) + value.size() for key, value in self.attrs.items()])
```

Comment:
By having the square brackets you will first materialize a list with the sizes of the keys and values of each item in `self.attrs`. By using round brackets instead you would have a generator which has lazy initialization. Python allows for dropping round brackets if they are immediately nested inside a function so could just drop square brackets.

This is more of a nit since there are limited amount of items in `self.attrs`
```suggestion
        return sum(bytesize(key) + value.size() for key, value in self.attrs.items())
```